docs/rbd-nbd.md

```diff
@@ -38,31 +38,39 @@ a host-path and the default log file path will be
 under the `cephLogDir` path on NodeStage(map) and removed the same on
 the respective NodeUnstage(unmap).
 
-In case if you need a customized log path, you should do below:
+- There are different strategies to maintain the logs
+  - `remove`: delete log file on unmap/detach (default behaviour)
+  - `compress`: compress the log file to gzip on unmap/detach
+  - `preserve`: preserve the log file in text format
 
-- Edit the daemonset templates to change the ceph log directory host-path
-  - If you are using helm charts, then you can use key `cephLogDirHostPath`
+  You can tweak the log strategies though `cephLogStatergy` option from the
```

Comment:
typo in `cephLogStatergy`

I always prefer to have Kubernetes components capitalized as they do in their docs, meaning StorageClass, DaemonSet and so on.